builders/build/substrate/customizing-chain-specs.md

```diff
@@ -0,0 +1,190 @@
+    "para_id": 3333,
+    // No need
+    "codeSubstitutes": {},
+    "genesis": { 
+        ... 
+    }
+}
+```
+
+The other important section of the file is within the `genesis` attribute, that contains the genesis state. In the following JSON snippet, the default values and configuration for some modules are presented as an example: 
+
+```json
+{
+    ...
+    "genesis": {
+        "runtime": {
+            ...
+            // Sets the account that will bear sudo privileges
+            "sudo": {
+                "key": "0xf24ff3a9cf04c71dbc94d0b566f7a27b94566cac"
+            },
+            // Sets the initial balances for some accounts
+            "balances": {
+                "balances": [
+                [
+                    "0x3cd0a705a2dc65e5b1e1205896baa2be8a07c6e0",
+                    1208925819614629174706176
+                ],
+                [
+                    "0x773539d4ac0e786233d90a233654ccee26a613d9",
+                    1208925819614629174706176
+                ],
+                [
+                    "0x798d4ba9baf0064ec19eb4f0a1a45785ae9d6dfc",
+                    1208925819614629174706176
+                ],
+                [
+                    "0xf24ff3a9cf04c71dbc94d0b566f7a27b94566cac",
+                    1208925819614629174706176
+                ]
+                ]
+            },
+            // Sets balances for EVM accounts
+            "evm": {
+                "accounts": {
+                "0x1000000000000000000000000000000000000001": {
+                    "nonce": "0x1",
+                    "balance": "0xd3c21bcecceda1000000",
+                    "storage": {},
+                    "code": [
+                    0
+                    ]
+                },
+                "0x6be02d1d3665660d22ff9624b7be0551ee1ac91b": {
+                    "nonce": "0x0",
+                    "balance": "0xffffffffffffffffffffffffffffffff",
+                    "storage": {},
+                    "code": []
+                },
+                "0xd43593c715fdd31c61141abd04a99fd6822c8558": {
+                    "nonce": "0x0",
+                    "balance": "0xffffffffffffffffffffffffffffffff",
+                    "storage": {},
+                    "code": []
+                }
+                }
+            },
+        }
+    }
+    ...
+}
+```
+
+## Generating a raw JSON Chain Specification File {: #generating-raw-specs-file }
+
+One final step before deploying the ContainerChain is converting the JSON specification file to a raw format, which is a compact, less-readable version of the same file, required to initialize a node.
+
+The following command will convert the chain specs file into the required raw format:
+
+```bash
+./target/release/container-chain-template-frontier-node \
+    build-spec --chain=chain_spec.json --raw > raw_chain_spec.json
+```
+
+And this file configured, customized, and in the correct raw JSON format now can be uploaded to initialize a new ContainerChain in Tanssi.
```

Comment:
```suggestion
Now that this file has been configured and customized and is in the correct raw JSON format, it can be uploaded to initialize a new ContainerChain in Tanssi.
```